### ISSUE TITLE
[USER-v2] Update Bitkeep wallet name and logo to Bitget Wallet

### ISSUE THREAD

**Issue by vmichalik** [superfluid-org/superfluid-dashboard#730]
## Overview
**What?**
You can find the logo here: https://drive.google.com/file/d/1kiQEE_hYcGVCsgJC9Q6b3_75XoQdv0qY/view?usp=sharing， Please confirm with us once it is done. We look forward to collaborating more in the future.
Please Notice that our brand name is Bitget Wallet (please do not only write Bitget to distinguish it from Bitget Exchange), and both words of the first letter should be capitalized “B” in Bitget and “W” in Wallet. Below are the correct examples for your reference. Thank you.

**How?**

**Why?**

**Deadline:**

**UI/UX Design:**

## Acceptance criteria
 - [ ] ...
 - [ ] Satisfies entire acceptance criteria
 - [ ] Consider communication & visibility
 - [ ] Code review is approved by at least 2 reviewers
 - [ ] Tested by Elvijs/QA & discovered bugs resolved
 - [ ] Documentation updated
 - [ ] Assignee/opened by merges PR
 - [ ] Automated E2E tests for happy path
 - [ ] Code is merged
 - [ ] Observability of critical user path
 - [ ] Steps to access code change as a user

## Links/Resources
 * ...

## Readiness criteria
- [ ] Descriptive title
- [ ] What?
- [ ] Why?
- [ ] How?
- [ ] Acceptance criteria complying with Definition of Done
- [ ] Estimate/Story points
- [ ] Deadline: If not a part of an epic
- [ ] Priority: Reflected as order on the board, walking the board from the top and right
- [ ] Assignee
- [ ] Labels
- [ ] Link to epic: If belonging to an epic
- [ ] A task is always a separate ticket/issue
- [ ] UI/UX Design (optional)
- [ ] Blockers/Prerequisites (optional): Use Zenhub blocked by/blocking feature
